Firebase/ios/Bonfire/Core/Channel/Channels.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+struct Channels {
```

Comment:
As well as being "good practice", turns our first class collections have the advantage of making Rx testing much easier. `Event` is only `Equatable` if the `Element` is also `Equatable`, and arrays can't be `Equatable` (being able to equate two arrays of `Equatable` objects doesn't mean it itself conforms to `Equatable`. First class collections, however, can be `Equatable`, which makes it much easier to verify the results of the test against the list of expected `Recorded<Event<Element>>`.

Also: `Equatable`.